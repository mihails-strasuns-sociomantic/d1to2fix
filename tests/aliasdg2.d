struct S
{
    alias void delegate ( int ) SomeDg;
}

void foo ( S.SomeDg dg1, S.SomeDg d2 );
