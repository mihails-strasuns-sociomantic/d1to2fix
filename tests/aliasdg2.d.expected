struct S
{
    alias void delegate ( int ) SomeDg;
}

void foo ( scope S.SomeDg dg1, scope S.SomeDg d2 );
